apiary.apib

```diff
@@ -3672,10 +3672,12 @@ Get details for a single work order.
                     + description: `Replacing pipework` (string)
             + materials (array)
                 + (object)
-                    + product (object)
+                    + product (object, nullable)
                         + type: `product` (string)
                         + id: `e2314517-3cab-4aa9-8471-450e73449040` (string)
                     + quantity: 3 (number)
+                    + name: `1/2" pipe` (string)
+                    + price: 30.20 (number)
```

Comment:
Money?

Reply:
Also; should we use `unit_price` like we do invoicing?

Reply:
We considered this, but since we're not accepting a currency, we didn't think it made sense here, right?

Reply:
Even if we don't accept it I think it's a good practice to be consistent in how we handle money values and currencies. I'm perfectly fine with throwing an exception if you try to pass a currency that is different from the department one. At least then the customer knows that it's not possible and we have a consistent api.